requirements.txt

```diff
@@ -3,3 +3,4 @@ geopandas
 rioxarray
 requests
 matplotlib
+ipykernel
```

Comment:
I added this here because to in the readme, we explain how to set up a jupyter notebook for pygcdl on ceres, and we instruct users to call both `pip install -r requirements.txt` and `pip install ipykernel`. I added ipykernel to requirements.txt to avoid having to call `pip install` twice, but then accidentally left the `pip install ipykernel` command in the readme. I agree that since it is not needed for the functionality of the package on its own, it shouldn't be included. Updated on fork.